includes/blocks.inc

```diff
@@ -106,6 +106,40 @@ function islandora_solr_block_info() {
 }
 
 /**
+ * Implements hook_block_configure().
+ */
+function islandora_solr_block_configure($delta = '') {
+  if ($delta == 'simple') {
+    $form = array();
+    $form['islandora_solr_simple_search_label_title'] = array(
+      '#type' => 'textfield',
+      '#title' => t('Simple Search Label'),
+      '#size' => 30,
+      '#default_value' => variable_get('islandora_solr_simple_search_label_title', 'Search Term'),
+      '#description' => t('Customize label text associated with search input form. Accessibility guidelines recommend whenever possible to explicitly associate label text with form inputs.'),
```

Comment:
@whikloj Mostly it's there to explain why the label is required in the first place combined with a checkbox (rather than just having a blank label, for example).